tests/test_sourceMatch.py

```diff
@@ -181,6 +181,7 @@ def testPhotometricCalib(self):
             s.setDec(dec * afwGeom.degrees)
             s.set(self.table.getPsfFluxKey(), psfMags[band])
 
+        ifd.close()
```

Comment:
Is it worth using the context manager? This is exactly what it's for.

    with open(os.path.join(afwdataDir, "CFHT", "D2", "sdss.dat"), "r") as ifd:
        allLines = ifd.readlines()
    ....
    for line in allLines:

Ditto below.